### PR TITLE
LPS-68418 buildLang

### PR DESCRIPTION
**Pull request by marcellustavares**
@vicnate5, @inacionery mention he wrote tests for this LPS.

thx

### CODE REVIEW

File: modules/apps/forms-and-workflow/dynamic-data-lists/dynamic-data-lists-form-web/src/main/java/com/liferay/dynamic/data/lists/form/web/internal/display/context/DDLFormAdminDisplayContext.java [hunk 1]
```diff
@@ -669,7 +665,7 @@ protected JSONObject toJSONObject(
 
 	private static final String[] _DISPLAY_VIEWS = {"descriptive", "list"};
 
-	private String _autosaveInterval;
+	private final int _autosaveInterval;
```
Comment (inacionery) [inline modules/apps/forms-and-workflow/dynamic-data-lists/dynamic-data-lists-form-web/src/main/java/com/liferay/dynamic/data/lists/form/web/internal/display/context/DDLFormAdminDisplayContext.java]: @marcellustavares, with this SF this variable is unnecessary

**Comment by marcellustavares** [comment]
Thanks for catching this Inacio, Victor could you please remove this
variable before forwarding to Brian?

Thx

Em 26 de out de 2016 19:06, "Inácio Nery" <notifications@github.com>
escreveu:

**Comment by vicnate5** [comment]
@marcellustavares Removed the extra variable.

**Comment by vicnate5** [comment]
Pushed to BChan (including test)
https://github.com/brianchandotcom/liferay-portal/pull/44342

**Pull request closed without merging**